README.md

```diff
@@ -53,12 +53,12 @@ Run `source environment`  now and whenever these environment files are modified.
 * Follow the instructions in https://cloud.google.com/sdk/downloads to get the `gcloud` command line utility.
 * In the [Google Cloud Console](https://console.cloud.google.com/), select the correct Google user account on the top
   right and the correct GCP project in the drop down in the top center. Go to "IAM & Admin", then "Service accounts",
-  then click "Create service account" and select "Furnish a new private key". Create the account and download the
+  then click "Create service account" and select "Furnish a new private key". Under "Roles" select "Project – Owner", "Project – Service Account Actor" and "Cloud Functions – Cloud Function Developer". Create the account and download the
```

Comment:
My hunch is that this is too permissive and, I think, should be scoped to the project via IAM but I don't know GCP well enough.